.github/workflows/build.yml

```diff
@@ -49,9 +41,9 @@ jobs:
     - name: Restore packages
       shell: bash
       run: |
+        ./pre-restore.sh
```

Comment:
Is calling this script still necessary? The props files are now included in git?

Reply:
yes, but only the props files not all the other scripts